src/CoreML/MLMultiArray.cs

```diff
@@ -15,8 +17,8 @@ namespace CoreML {
 	public partial class MLMultiArray {
 		static NSNumber[] ConvertArray (nint[] value)
 		{
-			if (value == null)
-				return null;
+			if (value is null)
+				return null!;
```

Comment:
I decided to make this one a `null!` instead of making the return here a `NSNumber []?` since the `ConvertArray` method gets called in the MLMultiArray constructors in lines 33 and 38 and according to those apple docs, those parameters cannot be null.

Reply:
Just return an empty array. You are just giving a perfect reason NOT to use !

Reply:
@mandel-macaque Good solution! Thanks